src/network/webtask.cpp

```diff
@@ -383,19 +383,33 @@ void WebTask::slotNetworkReplyFinished() {
 
     const auto statusCode = readStatusCode(*pFinishedNetworkReply);
     if (pFinishedNetworkReply->error() != QNetworkReply::NetworkError::NoError) {
-        onNetworkError(
-                pFinishedNetworkReply->error(),
-                pFinishedNetworkReply->errorString(),
-                WebResponseWithContent{
-                        WebResponse{
-                                pFinishedNetworkReply->url(),
-                                pFinishedNetworkReply->request().url(),
-                                statusCode},
-                        readContentType(*pFinishedNetworkReply),
-                        readContentData(pFinishedNetworkReply).value_or(QByteArray{}),
-                });
-        DEBUG_ASSERT(hasTerminated());
-        return;
+        if (checkIfTaskIsLooping() &&
+                pFinishedNetworkReply->error() ==
+                        QNetworkReply::NetworkError::ContentNotFoundError) {
+            kLogger.info()
+                    << this
+                    << "Task is Looping and received 404 in this request."
+                    << "This request is passed in order to get other results.";
```

Comment:
You may write: This error is not fatal. Are there other errors that are fatal / not fatal?

Reply:
As far as I tried there is none.

Reply:
I added the comment tho.